README

```diff
@@ -0,0 +1,37 @@
+loadflex: Models and Tools for Watershed Flux Estimates
```

Comment:
so there actually IS a readme, but it's called README.md now (because that gets prettily presented on the GitHub front page for the package). were you seeing errors because of the renaming?